### PR TITLE
Rules page

### PR DESCRIPTION
**Pull request by git-0r**
# created rules page screen for quiz app
quiz3

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+}
+
+.navbar-input {
+  width: 80%;
+  height: 30px;
```
Comment (manish-GH) [inline style.css]: use rem

**Review by manish-GH — APPROVED** [review]
Screenshot (5)
If you are working on the responsiveness, give it a min-height of 100vh and use flex so that the footer does not cross the bottom of the screen.

Otherwise, the code looks good. It can be merged

File: rules_page.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                <circle cx="12" cy="10" r="3" />
+                <circle cx="12" cy="12" r="10" />
+            </svg>
+        </div>
+    </nav>
+
+    <main class="main-content">
+
+        <div class="rules-container">
+            <h1>Rules:</h1>
```
Comment (varun1233) [inline rules_page.html:36]: Suggestion: You could have used li here.

**Review by varun1233 — APPROVED** [review]
Code Looks good. Just provide a small suggestion, apart from that everything is fine

**Review by gouravmarch20 — APPROVED** [review]

**Pull request merged**